### ISSUE TITLE
Update SDL2.dll to latest version

### ISSUE THREAD

**Issue by HordeResurrection** [stride3d/stride#1333]
**Is your feature request related to a problem? Please describe.**
I want to use `SDL_SetWindowMouseRect` and some other functions which is available since SDL 2.0.18
But Stride3D bundled with lower SDL version

**Describe the solution you'd like**
1. Download latest version from [libsdl.org](https://www.libsdl.org/download-2.0.php)
2. Place it in `Stride3D\deps\SDL2\`
3. Also you can to update [SDL2-CS](https://github.com/flibitijibibo/SDL2-CS)

**Describe alternatives you've considered**
I can update it manually in game build directory but it can cause unexpected behavior.

**Comment by HordeResurrection** [comment]
Does Stride developers have any plans to update the SDL version? @Eideren

**Comment by Eideren** [comment]
I don't believe any one in the core team is looking at this right now, feel free to make a PR about this :)

**Comment by HordeResurrection** [comment]
`sdl2.dll` shipped by `Silk.NET.SDL` in Stride now.
Unfortunately it's only `2.0.14` version
